lib/server.js

```diff
@@ -159,11 +161,11 @@ WalletService.getInstanceWithAuth = function(opts, cb) {
   var server = new WalletService();
   server.storage.fetchCopayerLookup(opts.copayerId, function(err, copayer) {
     if (err) return cb(err);
-    if (!copayer) return cb(new ClientError('NOTAUTHORIZED', 'Copayer not found'));
+    if (!copayer) return cb(new ClientError(Errors.codes.NOT_AUTHORIZED, 'Copayer not found'));
```

Comment:
Why we need to repeat 'Copay not found' string?

Cann't we do:
``` js
    if (!copayer) return cb(new Errors.codes.NOT_AUTHORIZED());
```
so something like that?